docs/how-to/load-validators-without-restarting.md

```diff
@@ -8,8 +8,8 @@ sidebar_position: 5
 
 You can load validators into a running Teku process, only if you started Teku by:
 
-- Specifying a directory using [`--validator-keys`](../reference/cli/index.md#validator-keys), or
-- Specifying a URL using [`--validators-external-signer-public-keys`](../reference/cli/index.md#validators-external-signer-public-keys).
+- specifying a directory using [`--validator-keys`](../reference/cli/index.md#validator-keys), or
```

Comment:
No initial UC?

Reply:
no, I think the rule is actually to capitalize if it is a full sentence of just a standalone item without a lead in sentence

Reply:
Please review the list guidelines from the Microsoft Style Guide: https://learn.microsoft.com/en-us/style-guide/scannable-content/lists. Items should always start with an uppercase letter, even if they're fragments, unless they start with a technical term that is always lowercase.

Reply:
Sorry @mjsmike62, you were absolutely correct. I didn't look hard enough at the style guide search for this. Thanks @alexandratran